### ISSUE TITLE
Cannot find module '@expo/image-utils'

### ISSUE THREAD

**Issue by jqtruong** [expo/expo-cli#755]
## Description

Just updated `expo-cli` and getting error in title when i try to run `expo start`.

```
$ expo start
Starting project at /...
Expo DevTools is running at http://localhost:19002
Press d to open DevTools now, or shift-d to always open it automatically.
Cannot find module '@expo/image-utils'
Error: Cannot find module '@expo/image-utils'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/@expo/xdl@55.0.3/src/AssetUtils.js:6:1)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
```

## Environment

```
$ EXPO_DEBUG=true expo diagnostics
Cannot read property '1' of null
TypeError: Cannot read property '1' of null
    at a.run.then.e (/home/nymo/.npm-global/lib/node_modules/expo-cli/node_modules/envinfo/dist/envinfo.js:1:97131)
```

**Comment by olee2002** [comment]
+1 About to post the same issue.  Everything was working great a few days ago.

Opening DevTools in the browser... (press shift-d to disable)
Cannot find module '@expo/image-utils'
Set EXPO_DEBUG=true in your env to view the stack trace.

Expo CLI 2.20.3 environment info:
    System:
      OS: macOS High Sierra 10.13.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.2 - /usr/local/bin/node
      Yarn: 1.15.2 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5199772
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^33.0.0 => 33.0.7
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz => 0.59.8
    npmGlobalPackages:
      expo-cli: 2.20.3

**Comment by JoeyKhd** [comment]
+1 on this one. 

Always returns @expo/image-utils not found. Fresh installation, popped up immediately.

**Comment by jesperjohansson** [comment]
I just got the same issue

**Comment by Nikit-Singh** [comment]
+1
I just installed expo-cli and then did `yarn start` and got this
`
yarn run v1.16.0
$ expo start
Starting project at /home/zappy/Documents/Code/MyApp
Expo DevTools is running at http://localhost:19002
Opening DevTools in the browser... (press shift-d to disable)
Cannot find module '/home/zappy/.npm-global/lib/node_modules/expo-cli/node_modules/@expo/image-utils/build/index.js'. Please verify that the package.json has a valid "main" entry
Set EXPO_DEBUG=true in your env to view the stack trace.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`

**Comment by juyod** [comment]
I fixed it by this:
```bash
npm install -g expo-cli@2.19.5
```

**Comment by Nikit-Singh** [comment]
This issue occurs after updating to `expo-cli@2.20.0` or higher.

**Comment by vigneshraaj** [comment]
The workaround,  of downgrading expo-cli to v2.19.5 is not working with sdk v33+

![Screenshot_1561400651](https://user-images.githubusercontent.com/2261556/60043307-66507b00-96dd-11e9-8580-b15719824d66.png)

**Comment by eugenegodun** [comment]
@vigneshraaj Downgrading to `expo-cli@2.19.5` works for me ok along with `sdk v33`. Check if you've migrated correctly from `sdk 32` to `sdk 33`. https://blog.expo.io/expo-sdk-v33-0-0-is-now-available-52d1c99dfe4c

**Comment by cruzach** [comment]
Hey everyone, just wanted to say we're aware of this and currently looking into it. @vigneshraaj- I don't believe that your issue is related

**Comment by cruzach** [comment]
Fixed in expo-cli@2.20.5! 🎉

Thanks everyone for letting us know about this issue so quick, and being patient for the fix 😄
